xdsl/interactive/app.py

```diff
@@ -107,10 +107,10 @@ class InputApp(App[None]):
     """
 
     INITIAL_IR_TEXT = """
-        func.func @hello(%n : index) -> index {
-          %two = arith.constant 2 : index
-          %res = arith.muli %n, %two : index
-          func.return %res : index
```

Comment:
Is there a reason for these changes? Those look like xdsl-specific and not textual specific. How did this have an impact?

Reply:
So sorry - this is a spastic error I have no clue how it got in.

Reply:
removed it.